### ISSUE TITLE
JetBrains support

### ISSUE THREAD

**Issue by win4r** [saoudrizwan/claude-dev#63]
https://youtu.be/Us6LQzKmgfs

**Comment by saoudrizwan** [comment]
Cool video! I don't have any plans to add JetBrains support for now, but am willing to look into it if enough people want it

**Comment by sergeyka** [comment]
+1 for the jetbrains support

**Comment by kvenLin** [comment]
+1 for the jetbrains support

**Comment by atoptun** [comment]
+100 for the jetbrains support

**Comment by ribber** [comment]
+1 for the jetbrains support

**Comment by cpatrickalves** [comment]
+1 for the jetbrains support

**Comment by hedward** [comment]
+1 for the jetbrains support

**Comment by AllyourBaseBelongToUs** [comment]
Since this is open.

Bump for both IDE´s

**Comment by Kusnezow94** [comment]
Bumb for PyCharm

**Comment by jgwconsulting** [comment]
+1 for JetBrains :)

**Comment by AlexJ-StL** [comment]
+1 for JetBrains/PyCharm

**Comment by benjamix95** [comment]
+1 for JetBrains/PyCharm

**Comment by metal450** [comment]
+1 for Jetbrains/Webstorm

**Comment by kevin00730** [comment]
+1 for JetBrains

**Comment by jakubkalicki** [comment]
+1

**Comment by jasonoh** [comment]
+1 for PyCharm & JetBrains

**Comment by jmartsch** [comment]
+1 JetBrains IntelliJ

**Comment by msti** [comment]
+1

**Comment by RobinGasi** [comment]
+1

**Comment by edaneshi** [comment]
Zed!

**Comment by alexn-tinwell** [comment]
+1 for JetBrains; their own AI is one of the least effective and GitHub Copilot is far behind your work

**Comment by ronalson** [comment]
`zed` support would be awesome.

**Comment by shiftlabs1** [comment]
+1 for jetbrains

**Comment by AlexJ-StL** [comment]
> Zed!

I 2nd the inclusion of Zed, but only after JetBrains because I have to f-ing compile Zed on W11 and/or WSL2 every time I run it. That said, Zed is pretty cool so far.

**Comment by spikeliu1984** [comment]
+1 for the JetBrains support

**Comment by CoyoteLeo** [comment]
+1 for the JetBrains!

**Comment by africa1207** [comment]
+1 for Jetbrains/Webstorm

**Comment by kamjin1996** [comment]
+100 for the jetbrains support.
I'm still using the 2021 version of idea, which is stable and excellent!

**Comment by Kansai53** [comment]
+1 for Jetbrains Webstorm! Wish I could use this, but Webstorm is so much better than VSCode so I'm stuck without Claude-dev

**Comment by itbeard** [comment]
+1 for Rider (JetBrains)

**Comment by luyi404** [comment]
Jetbrains! Clion

**Comment by SumeraMartin** [comment]
+1 Jetbrains support (Android Studio)

**Comment by Ricent82** [comment]
Jetbrains Webstorm & Rider pleeeeeeassseee

**Comment by ibrahimel** [comment]
+1 for Jetbrains !

**Comment by cosmobet** [comment]
+1 for JetBrains PyCharm :)

**Comment by daniel23972** [comment]
+1 for the jetbrains support

**Comment by metal450** [comment]
> but am willing to look into it if enough people want it

Wondering how many people would be considered enough @saoudrizwan ? Sure seems like there's tons who want it ;)

**Comment by thomastrinn** [comment]
+1 for Jetbrains IntelliJ

**Comment by leaanthony** [comment]
+1 for jetbrains!

**Comment by Motrotz** [comment]
Since people are constantly commenting +1, wouldn't it make sense to just keep one comment per IDE and assess demand by upvotes?  I personally use VS Code for quick changes or small projects, but VS2022 or Android Studio for bigger tasks and as such would profit from both VS2022 and JetBrains plugin versions.

**Comment by Benjoyo** [comment]
Many plugins are just for VSCode despite its low popularity in the huge JVM ecosystem etc. Please Jetbrains 🙏

**Comment by SHJordan** [comment]
+1 for PHPStorm(JETBRAINS)

**Comment by benoitjadinon** [comment]
please vote for IntelliJ on #581

**Comment by maryasov** [comment]
+1 for JetBrains

**Comment by NakiriYuuzu** [comment]
+1 for jetbrians/*

**Comment by anibalinbalin** [comment]
jetbrains/webstorm! they just announced its free for non comercial usage! thanks!

**Comment by Loui2** [comment]
+1 Jetbrains

**Comment by scofield-hello** [comment]
+1 Jetbrains

**Comment by Motrotz** [comment]
> Since people are constantly commenting +1

This is still happening, spamming the issue without providing any value at at all. this only leads to people unfollowing / muting the discussion.


> just keep one comment per IDE and assess demand by upvotes

this is something You can do yourself, by just clicking "thumbs up" the first time you see your IDE of choice mentioned. But since this is not how it's done right now, it might be worth a shot for you @saoudrizwan to chime in and hide / delete all the spam comments, so that everyone coming into the discussion can easily vote for their IDE of choice. Alternatively, you might want to create a new issue, providing comments to upvote himself. I don't know what's best, and I don't want to judge your decisions. What i will do is remove this issue from my watchlist if the +1 spam comments continue.

To everyone else: please feel encouraged to comment here to have a healthy discussion. But think twice before posting spammy comments that could've been just a reaction. Thank you.

**Comment by openhollywood** [comment]
+∞ for JetBrains

**Comment by ruze00** [comment]
+ about at least 25 for all the teams I work with

**Comment by 534002646** [comment]
jetbrains +1

**Comment by stekow** [comment]
+1

**Comment by sachicali** [comment]
+1

**Comment by Motrotz** [comment]
![image](https://github.com/user-attachments/assets/bdf8d02b-2f26-4e31-b9a5-9444eca0dbbe)
If your comment get's hidden like this, you're doing something wrong @sachicali ...

**Comment by michabbb** [comment]
@all people posting "+1" here instead of simply clicking 👍 on the original post: HOW BRAIN-DEAD ARE YOU???  
Some might think developers are smart—guess that's just a superstition. 🙄🙈  
You brilliant sheep out there are forcing others to mute this thread. Great job!

**Comment by FTDRTD** [comment]
Visual Studio 2022 +1

**Comment by FTDRTD** [comment]
Please provide support for extensions for Visual Studio 2022

**Comment by guixin567** [comment]
+1 jetbrains

**Comment by zxq9133** [comment]
good idea to support jetbrain IDEs. may be we can refer the architecture of [continue](https://github.com/continuedev/continue). The core and WebView-related module code can be shared by both IDEs.

**Comment by Bratzmeister** [comment]
+1 for jetbrains (PyCharm, WebStorm, ...)

**Comment by zinkls** [comment]
+1 for jetbrains IDEs

**Comment by bllooi** [comment]
+1 for Jetbrains

**Comment by zhanglei-workspace** [comment]
hi ,when is  it come true?

**Comment by coderberry** [comment]
+1

**Comment by awsfanatic** [comment]
+1

**Comment by YJL626** [comment]
+1 for JetBrains

**Comment by timerzz** [comment]
+1

**Comment by bllooi** [comment]
It looks like the team is aware that alot of people want this and they are hiring for a lead of the project at the moment.

https://cline.bot/join-us

```
Software Engineer, Kotlin

Drive JetBrains plugin development and lead our open source community

🌉 Location: San Francisco, CA (On-site)

💰 Compensation: $150K-250K + equity as founding team member

You'll own the development of our JetBrains plugin, working in Kotlin to bring our autonomous AI assistant to millions of developers who prefer IntelliJ, PyCharm, and other JetBrains IDEs. While the core of our extension lives in a shared TypeScript/React codebase, you'll build the critical Kotlin infrastructure that interfaces with JetBrains' platform APIs and our shared components.
Key Responsibilities:

    Lead development of our JetBrains plugin infrastructure in Kotlin
    Design and implement interfaces between Kotlin and our shared TypeScript/React codebase
    Build performant plugin features using JetBrains Platform SDK
    Collaborate with our TypeScript team on shared architecture decisions
    Help define best practices for cross-IDE compatibility
    Shape our technical roadmap and engineering culture
    Review PRs and issues from our open source community

Required Qualifications:

    BS in Computer Science or related field
    3+ years of Kotlin development experience (Android or backend)
    Strong understanding of JVM ecosystem
    Experience with TypeScript and React

Nice to Have:

    Experience building JetBrains plugins
    Experience with IntelliJ Platform SDK
    Background in developer tools or IDE extensions
    Open source contributions
    Experience with prompt engineering and LLM integration

What We Offer:

    Competitive salary: $150K-250K
    Significant equity as founding team member
    Full medical, dental, and vision insurance
    Relocation assistance
    Direct impact on product architecture and company direction
    Learning and conference budget
    Latest development hardware and tools

Our Stack:

    Kotlin for JetBrains plugin implementation
    TypeScript/Node.js for shared extension logic
    React for webview interfaces
```

**Comment by redpintings** [comment]
Support adding pycharm+ one

**Comment by fanpei91** [comment]
+1 for JetBrains

**Comment by chinese-wzq** [comment]
+1 for JetBrains

**Comment by 524994432** [comment]
+1 for JetBrains

**Comment by Nopik** [comment]
+1 for JetBrains

**Comment by yuanliangxie** [comment]
+1 for JetBrains
